src/components/votes/VoteCard.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+	import DirectionStraightRight from 'carbon-icons-svelte/lib/DirectionStraightRight.svelte';
+	import { DefaultVotingResult, type Voting } from '$models/voting';
+	import { Tag } from 'carbon-components-svelte';
+	import dayjs from 'dayjs';
+	import 'dayjs/locale/th';
+	import buddhistEra from 'dayjs/plugin/buddhistEra';
+	import type { VoteCardProps } from '../../routes/assemblies/[id]/+page';
+
+	dayjs.extend(buddhistEra);
+	dayjs.locale('th');
+
+	interface VoteCardTheme {
+		bg: string;
+		hoveredBg: string;
+		tagBg: string;
+		tagFontColor: string;
+	}
+	const CARD_THEMES: Record<DefaultVotingResult, VoteCardTheme> = {
+		[DefaultVotingResult.Passed]: {
+			bg: 'bg-teal-10',
+			hoveredBg: 'hover:bg-teal-20',
+			tagBg: 'bg-teal-30',
+			tagFontColor: 'text-text-01'
+		},
+		[DefaultVotingResult.Failed]: {
+			bg: 'bg-red-10',
+			hoveredBg: 'hover:bg-red-20',
+			tagBg: 'bg-red-30',
+			tagFontColor: 'text-text-01'
+		}
+	};
+	const CANDIDATE_CARD_THEME: VoteCardTheme = {
+		bg: 'bg-purple-10',
+		hoveredBg: 'hover:bg-purple-20',
+		tagBg: 'bg-purple-70',
+		tagFontColor: 'text-white'
+	};
+
+	export let voting: VoteCardProps['voting'] = {} as Voting;
+	export let highlightedVoteByGroups: VoteCardProps['highlightedVoteByGroups'] = [];
+
+	interface HighlightedVoteSummary {
+		totalCount: number;
+		totalAmount: number;
+	}
+	$: ({ totalCount, totalAmount } = highlightedVoteByGroups.reduce<HighlightedVoteSummary>(
+		reduceHighlightedVoteSummary,
+		{ totalCount: 0, totalAmount: 0 }
+	));
+	$: theme = CARD_THEMES[voting.result as DefaultVotingResult] || CANDIDATE_CARD_THEME;
+	$: isCandidate = ![DefaultVotingResult.Failed, DefaultVotingResult.Passed].includes(
+		voting.result as DefaultVotingResult
+	);
+
+	function reduceHighlightedVoteSummary(
+		{ totalAmount, totalCount }: HighlightedVoteSummary,
+		{ count, total }: VoteCardProps['highlightedVoteByGroups'][number]
+	): HighlightedVoteSummary {
+		return {
+			totalCount: totalCount + count,
+			totalAmount: totalAmount + total
+		};
+	}
+</script>
+
+<div
+	class={`vote-card rounded-sm relative p-4 flex flex-col gap-y-2 w-72 h-64.5 whitespace-break-spaces ${theme.bg} ${theme.hoveredBg}`}
+>
+	<p class="body-compact-01 text-text-02">
+		{dayjs(voting.date).format('D MMM BB')}
+	</p>
+	<a class="vote-card--url after:inset no-underline after:absolute w-56" href={voting.sourceUrl}>
```

Comment:
My bad for not clarifying this, the `sourceUrl` is the original document link for each voting from the parliament. The link from the card should refer to the voting page that can be referred to with the `id`. However, we don't have a page for voting yet, you can just use static `#` for now (the Svelte build will fail if we add a 404 path). And `sourceUrl`  needed to be removed from the type and mock data.